### PR TITLE
LRAC-14547

### PR DESCRIPTION
**Pull request by riccardo-ferrari**
Hi team,

In this PR:

- We include the `publishedDXPVariantId` for TERMINATED tests when sending a notification to AC.
- Update unit tests to validate the util class changes.

Thanks!

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by riccardo-ferrari** [comment]
ci:test:sf

**Comment by riccardo-ferrari** [comment]
ci:test:relevant

File: modules/dxp/apps/segments/segments-asah-connector/src/test/java/com/liferay/segments/asah/connector/internal/client/model/util/ExperimentUtilTest.java [hunk 1]
```diff
@@ -216,6 +216,95 @@ public void testToExperimentWithSegmentsExperimentWithStatusCompleted()
...
+			SegmentsExperimentConstants.Goal.BOUNCE_RATE.getLabel(),
+			SegmentsExperimentConstants.STATUS_TERMINATED);
+
+		String channelId = RandomTestUtil.randomString();
+		String dataSourceId = RandomTestUtil.randomString();
+		String pageURL = RandomTestUtil.randomString();
+
+		Experiment experiment = ExperimentUtil.toExperiment(
+			channelId, dataSourceId, RandomTestUtil.randomString(), layout,
+			LocaleUtil.ENGLISH, pageURL, _segmentsEntryLocalService,
```
Comment (ealonso) [inline modules/dxp/apps/segments/segments-asah-connector/src/test/java/com/liferay/segments/asah/connector/internal/client/model/util/ExperimentUtilTest.java:260]: @riccardo-ferrari I think we can reuse here locale, right?

**Comment by ealonso** [comment]
Hey @riccardo-ferrari,

I've just added a small change, when you change it, you can forward it if you want.

thanks!!

File: modules/dxp/apps/segments/segments-asah-connector/src/test/java/com/liferay/segments/asah/connector/internal/client/model/util/ExperimentUtilTest.java (hunk 1 above)
Comment (riccardo-ferrari) [reply modules/dxp/apps/segments/segments-asah-connector/src/test/java/com/liferay/segments/asah/connector/internal/client/model/util/ExperimentUtilTest.java]: Thanks @ealonso !
I've fixed it on the `testToExperimentWithSegmentsExperimentWithStatusCompleted` test too.

**Comment by riccardo-ferrari** [comment]
ci:forward:force

**Comment by ealonso** [comment]
ci:forward

**Comment by riccardo-ferrari** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**